app/src/index.html

```diff
@@ -11,8 +11,8 @@
   <!-- Facebook OG tags -->
   <meta property="og:title" content="Academix | Sustainable Education Foundation"/>
   <meta property="og:type" content="website"/>
-  <meta property="og:url " content="https://sefglobal.org/academix/"/>
-  <meta property="og:image" content="https://sefglobal.org/academix/img/academix-logo.png"/>
+  <meta property="og:url" content="https://sefglobal.org/academix/"/>
```

Comment:
I think better to remove OG URL since the current URL is self-descriptive.

Reply:
Done